src/components/Footer/Footer.tsx

```diff
@@ -30,7 +30,7 @@ const Footer = () => (
             >
               Powered by
             </p>
-            <a href="https://www.titans-rugby.com/">
+            <a href="https://www.titans-rugby.com/" target="_blank" rel="noreferrer">
```

Comment:
Do we need the "noreferrer"? I'd have thought there's no issue with allowing analytics to see if traffic coming to RT or AND websites are from Cookie's Khaw?

Reply:
ESLint didn't like it without a noreferrer as it's vulnerable on older browsers.